src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -64,6 +77,11 @@ internal override sealed JsonParameterInfo CreateJsonParameterInfo()
             return new JsonParameterInfo<T>();
         }
 
+        internal override sealed JsonConverter<TargetType> CreateCastingConverter<TargetType>()
```

Comment:
Seems to be breaking the build
```suggestion
        internal sealed sealed JsonConverter<TargetType> CreateCastingConverter<TargetType>()
```